.github/workflows/deepx-ci.yml

```diff
@@ -0,0 +1,28 @@
+name: DeepX CI
+
+run-name: Build Docker container and perform unit testing
+
+on: [push, workflow_dispatch]
+
+jobs:
+  build-library:
+    runs-on: ubuntu-20.04
+    name: Build and test
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - name: Login to Github Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: deepx-inc
+          password: ${{ github.token }}
+
+      - name: Build and commit the Docker container
+        run: |
+          docker build --build-arg BASE_IMAGE=ghcr.io/deepx-inc/base_images --build-arg TAG=foxy -t deepx-gz-rendering .
```

Comment:
Based on Kristjan's comment, we don't need `--build-arg` for `docker build`.

Reply:
Changed to:

```
docker run deepx-gz-rendering:latest bash \
    -c 'cd /root/gazebo/gz-rendering/build; \
    cmake ..; \
    cmake --build . --target install -j; \
    ctest;'
 ```

Reply:
Removed `-j` as it made cmake build not finished. So reverted it to serial build.